orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -5905,9 +5905,9 @@ def generate_inputs(batch_size, max_seq_length, vocab_size):
     assert len([node.op_type for node in training_model.graph.node if node.op_type == "Squeeze"]) == 1
     assert len([node.op_type for node in training_model.graph.node if node.op_type == "PadAndUnflatten"]) == 1
     if case >= 2:
-        assert len([node.op_type for node in training_model.graph.node if node.op_type == "ShrunkenGather"]) == 2
+        assert len([node.op_type for node in training_model.graph.node if node.op_type == "FlattenAndUnpad"]) == 3
```

Comment:
why the number changed from 2 to 3?

Reply:
there is another one of "FlattenAndUnpad" as Gradient of "PadAndUnflatten".

Reply:
there is another one of "FlattenAndUnpad" as Gradient of "PadAndUnflatten".